vassal-app/src/main/resources/VASSAL/i18n/VASSAL.properties

```diff
@@ -75,6 +75,9 @@ Dialogs.out_dated_usage=This module contains custom code that is calling into VA
 Dialogs.incompatible.title=Incompatible Custom Code
 Dialogs.incompatible.heading=The Custom Code In This Module Should Be Recompiled
 Dialogs.incompatible.message=This module contains custom Java code (%1$s) which was not compiled to be Java 11 compatible. As a result, this module will not run on all versions of Java which VASSAL itself supports.
+Dialogs.module_version_incompatible=File Written by Different Version of Module
+Dialogs.module_version_incompatible_sequence=The file you are attempting to load may have been written by a different version of this module, and your version is therefore unable to find some of the data it is looking for. Continuing with this version may not faithfully represent the complete game state. 
+Dialogs.check_for_updated_vmod=Please check whether there is an updated version of this module (or if on the other hand your opponent needs an update)
```

Comment:
I would just make that last line:
'Please check check that you and your opponent are using the same version of the module.'

Reply:
Otherwise good to go.